deployment/step1.md

```diff
@@ -0,0 +1,41 @@
+# Environment Setup for Cloud Node
+
+This step basically sets up the environment required for deploying kubeedge.
+
+Do kubeadm init
+
+`kubeadm init --kubernetes-version $(kubeadm version -o short) --pod-network-cidr=10.244.0.0/16`{{execute HOST1}}
```

Comment:
Does katacoda already have a k8s cluster running ? If yes, can we use it ?

Reply:
Cluster is available but nodes are not configured. To configure the nodes we need to run kubeadm. Below is the instruction they have provided.
![clusterinstn](https://user-images.githubusercontent.com/14183934/57828254-86c91380-77c8-11e9-92ff-77f7ad0e2c9c.png)

Reply:
OK, Sounds good